build.yaml

```diff
@@ -574,7 +547,7 @@ common_cli:
   build_dockerfile_centos7: &build_dockerfile_centos7 |
     # Download the latest pip requirements file from flocker
     wget -c \
-    https://raw.githubusercontent.com/ClusterHQ/flocker/${TRIGGERED_BRANCH}/requirements.txt
+    https://raw.githubusercontent.com/ClusterHQ/flocker/${TRIGGERED_BRANCH}/requirements/admin.txt
```

Comment:
This may not be quite enough. 
The admin scripts import stuff (`rpm_version`) from flocker so we probably need to install requirements/flocker.txt too.